tools/resourcedocsgen/cmd/metadata.go

```diff
@@ -420,6 +369,15 @@ func isComponent(keywords []string) bool {
 	return getTagFromKeywords(keywords, "kind/component") != nil
 }
 
+func isFeaturedPackage(str string) bool {
+	for _, v := range featuredPackages {
+		if v == str {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Nit: Can we just use https://pkg.go.dev/golang.org/x/exp/slices#Contains here?

```suggestion
func isFeaturedPackage(package string) bool {
	return slices.Contains(featuredPackages, package)
}
```

Reply:
Sure thing!